src/EnergyPlus/FaultsManager.cc

```diff
@@ -1965,6 +2061,62 @@ namespace FaultsManager {
         return UAReductionFactorAct;
     }
 
+    Real64 FaultPropertiesFoulingCoil::FaultFraction()
+    {
+        // SUBROUTINE INFORMATION:
+        //       AUTHOR         Julien Marrec, EffiBEM
+        //       DATE WRITTEN   Feb. 2020
+
+        // PURPOSE OF THIS SUBROUTINE:
+        // Calculate the Fault Fraction based on Availability and Severity Schedules
+
+        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
+        Real64 FaultFrac(0.0); // Fault Fraction
+
+        // Check fault availability schedules
+        if (ScheduleManager::GetCurrentScheduleValue(this->AvaiSchedPtr) > 0.0) {
+
+            // Check fault severity schedules (Ptr initialized to -1, so would return a FaultFrac of 1 if not set)
+            FaultFrac = ScheduleManager::GetCurrentScheduleValue(this->SeveritySchedPtr);
+        }
+
+        return FaultFrac;
+    }
+
+    //Real64 FaultPropertiesFoulingCoil::CalFaultyCoilFoulingFactor()
+    //{
+        //// SUBROUTINE INFORMATION:
+        ////       AUTHOR         Julien Marrec, EffiBEM
+        ////       DATE WRITTEN   Feb. 2020
+
+        //// PURPOSE OF THIS SUBROUTINE:
+        //// Calculate the fouling thermal insulance factor (the reciprocal of a heat transfert coefficient) due to fouling in a coil
+
+        //// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
+        //Real64 FoulingFactor(0.0); // Fouling Thermal Insulance Factor (K/W)
+
+        //Real64 FaultFrac = this->FaultFraction();
+        //// If severity is zero, there's no fouling
+        //if (FaultFrac > 0.0) {
+
+            //// If "FouledUARated", and coil is autosized, this requires sizing, so I cannot store the UA of the coil on this Fault object here
+            //// I have to use an index
+            //if (this->FoulingInputMethod == iFouledCoil_UARated) {
+                //if (this->FouledCoiledType == WaterCoils::WaterCoil_SimpleHeating) {
+                    //FoulingFactor = FaultFrac * (1.0 / this->UAFouled - 1.0 / WaterCoils::WaterCoil(this->FouledCoilNum).UACoil);
+                //} else { // WaterCoils::WaterCoil_Cooling
+                    //FoulingFactor = FaultFrac * (1.0 / this->UAFouled - 1.0 / WaterCoils::WaterCoil(this->FouledCoilNum).UACoilTotal);
+                //}
+            //} else {
+                //// This case is simpler, doesn't require knowledge of the Coil object
+                //FoulingFactor = FaultFrac * (this->Rfw / (this->Aratio * this->Aout) + this->Rfa / this->Aout);
+            //}
+        //}
+
+        //// Do not allow improving coil performance
+        //return max(FoulingFactor, 0.0);
+    //}
```

Comment:
Was this just accidentally left in?

Reply:
Yes. 
Originally I had made that it's own routine and calculated here, which I liked better (Fault stuff contained in the Fault class itself)

But then I realized when checking and modifying the Cooling coil one that it definitely wasn't as simple and couldn't be well contained, so I resolved to move the calculation of the UAs (UAInternal / UAExternal) in the WaterCoils.cc  directly...

Reply:
No problem.  I have my local commit ready with these commented lines deleted.